galaxy.yml

```diff
@@ -9,7 +9,7 @@ namespace: pulp
 name: pulp_installer
 
 # The version of the collection. Must be compatible with semantic versioning
-version: 3.3.1
+version: 3.4.0
```

Comment:
@dkliban We should bump this to 3.5.0dev though. Just be aware that when w do a 3.4.1 release, we'd have to set it to back to 3.4.1.